packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -336,9 +334,10 @@ class ScaffoldState extends State<Scaffold> {
       ));
     }
 
-    return new CustomMultiChildLayout(children, delegate: _scaffoldLayout);
+    return new CustomMultiChildLayout(children, delegate: new _ScaffoldLayout());
   }
 
+  bool shouldRelayout(MultiChildLayoutDelegate oldDelegate) => true;
```

Comment:
no i mean this getter is just on the wrong class entirely. it's on the scaffold rather than the custom layout guy.